index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React + TS</title>
   </head>
-  <body>
+  <body style=margin:0>
```

Comment:
```suggestion
  <body style="margin: 0">
```

styles should always be surrounded by "